applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DFU_MCUMGR_ENABLE
+	bool "Enable DFU using MCUmgr"
+	select CAF_BLE_SMP_TRANSFER_EVENTS if MCUMGR_TRANSPORT_BT
+	depends on !CAF_BLE_SMP
+	depends on MCUMGR_MGMT_NOTIFICATION_HOOKS
```

Comment:
Consider using `select` instead of `depends on` to simplify configuration of the module (allow user to enable it with a single Kconfig).